src/System.Security.Cryptography.Algorithms/src/Resources/Strings.resx

```diff
@@ -79,6 +79,9 @@
   <data name="Arg_CryptographyException" xml:space="preserve">
     <value>Error occurred during a cryptographic operation.</value>
   </data>
+  <data name="Cryptography_BadData" xml:space="preserve">
+    <value>Bad Data.</value>
```

Comment:
So, I have no plans right now to alter what message gets shown on Windows (which is, in fact "Bad Data.".  But I can alter this one, since there's been two different comments about it.